WrightTools/_group.py

```diff
@@ -178,21 +179,113 @@ def parent(self):
             assert self._parent is not None
         except (AssertionError, AttributeError):
             from .collection import Collection
-            name = super().parent.attrs['name']
-            self._parent = Collection(self.filepath, name=name, edit_local=True)
+            key = posixpath.dirname(self.fullpath) + posixpath.sep
+            self._parent = Collection.instances[key]
         finally:
             return self._parent
 
     def close(self):
         """Close the group. Tempfile will be removed, if this is the final reference."""
         if(self.fid.valid > 0):
-            self.__class__.instances.pop(self.fullpath)
-            self.file.flush()
-            self.file.close()
-            if hasattr(self, '_tmpfile'):
-                os.close(self._tmpfile[0])
-                os.remove(self._tmpfile[1])
+            self.__class__.instances.pop(self.fullpath, None)
+            try:
+                self.file.flush()
+                self.file.close()
+            except SystemError:
+                pass
+            finally:
+                if hasattr(self, '_tmpfile'):
+                    os.close(self._tmpfile[0])
+                    os.remove(self._tmpfile[1])
+
+    def copy(self, parent=None, name=None, verbose=True):
```

Comment:
I'm slightly confused as to why we are not just relying on the underlying h5py `copy`
http://docs.h5py.org/en/latest/high/group.html#Group.copy

It seems to have most of what we want, except creation of a temp file. It recursively copies the content, including attributes

Reply:
natural name, as well, I suppose,

But, the inner workings I feel could be replaced by a call to `super` and adding the kwargs in that the underlying method supports, perhaps